channels/alpha

```diff
@@ -7,6 +7,8 @@ spec:
     - name: kope.io/k8s-1.5-debian-jessie-amd64-hvm-ebs-2017-01-09
       providerID: aws
       kubernetesVersion: ">=1.5.0"
+    - providerID: gce
+      name: "cos-cloud/cos-stable-56-9000-84-2"
```

Comment:
On GCE, it should be the publicly available image.

```
> gcloud compute images list
NAME                                              PROJECT                  FAMILY                    DEPRECATED  STATUS
...
debian-8-jessie-v20170308                         debian-cloud             debian-8                              READY
cos-beta-58-9334-19-0                             cos-cloud                cos-beta                              READY
cos-dev-59-9394-0-0                               cos-cloud                cos-dev                               READY
cos-stable-56-9000-84-2                           cos-cloud                cos-stable                            READY
cos-stable-57-9202-56-0                           cos-cloud                cos-stable                            READY
rhel-6-v20170227                                  rhel-cloud               rhel-6                                READY
...
```

Not sure if I opted into something to get those to show up in my account though.  Do you have them?